test/integration/datasources_k8s.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats
+
+load helper
+
+function setup () {
+  mkdir -p /var/run/secrets/kubernetes.io/serviceaccount
+  touch /var/run/secrets/kubernetes.io/serviceaccount/token
+  cp /tmp/ca.crt  /var/run/secrets/kubernetes.io/serviceaccount/
```

Comment:
This `/tmp/ca.crt` file won't exist when these tests are run outside of a container - would a relative path work?

Reply:
see the next comment about this. I will try...